src/shared/components/event-management/components/task-assigned/task-assigned.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="govuk-!-margin-9">
+  <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="error-summary">
+    <h2 class="govuk-error-summary__title" id="error-summary-title">
+      There is a problem
+    </h2>
+    <div class="govuk-error-summary__body">
+      <ul class="govuk-list govuk-error-summary__list">
+        <li>
+          <a href="#">Task assignment required</a>
+        </li>
+      </ul>
+    </div>
+  </div>
+
+  <div class="govuk-form-group govuk-form-group--error">
+      <h2 class="govuk-heading-m">Task assignment required</h2>
+
+    <p>This task is assigned to William Priest. You must assigned it to yourself to continue.</p>
```

Comment:
Query - Why William Priest? Can we just say 'another user' for now?

Reply:
I removed the hard-coding, this component will be updated when working on further jira tasks related to task event completion epic.